app/views/providers/means_reports/show.html.erb

```diff
@@ -9,9 +9,11 @@
 
   <%= render(
         'shared/check_answers/client_details',
-        attributes: %i[first_name last_name date_of_birth age national_insurance_number],
+        attributes: %i[first_name last_name date_of_birth age means_test national_insurance_number],
         applicant: @legal_aid_application.applicant,
         read_only: true) %>
 
-  <%= render "with_cfe_result_details" %>
+  <% if !@legal_aid_application.non_means_tested? %>
+    <%= render "with_cfe_result_details" %>
```

Comment:
To slightly extend Connor's suggestion, how about, as it's a oneliner
```suggestion
    <%= render "with_cfe_result_details" unless @legal_aid_application.non_means_tested? %>
```

Reply:
Thanks @colinbruce @cpjmcquillan I agree and have implemented this suggestion